test/e2e/options.booleanSchemas.ts

```diff
@@ -0,0 +1,10 @@
+export const input = {
+  properties: {
+    always: true,
+    never: false
```

Comment:
Can you expand these tests to:

1. Verify the output when the `unknownAny` option is `true` and `false` (as I noted above).
2. Verify the output when a sub-schema is `true` or `false` (try to cover at least a nested property and a definition).
3. Verify the output when the top-level schema itself is `true` or `false` (ie. `export const input = true`, `export const input = false`).